doc/user/content/sql/create-source/postgres.md

```diff
@@ -123,6 +124,30 @@ CREATE SOURCE mz_source
   WITH (SIZE = '3xsmall');
 ```
 
+### Monitoring source progress
+
+By default, PostgreSQL sources expose progress metadata as a subsource that you
+can use to monitor source **ingestion progress**. The name of the progress
+subsource can be specified when creating a source using the `EXPOSE PROGRESS
+AS` clause; otherwise, it will be named `<src_name>_progress`.
+
+The following metadata is available for each source as a progress subsource:
+
+Field          | Type                                     | Meaning
+---------------|------------------------------------------|--------
+`lsn`          | [`uint8`](/sql/types/uint/#uint8-info)   | The last Log Sequence Nubmer (LSN) consumed from the upstream PostgreSQL replication stream.
+
+And can be queried using:
+
+```sql
+SELECT lsn
+FROM <src_name>_progress;
+```
+
+As long as the LSN continues increasing, Materialize is consuming change data
```

Comment:
I don't know if we want to write anything more here but this may imply that if the LSN is not increasing something might be going wrong, which is not the case, it could just be that the upstream database has no changes.

Reply:
Ah, true! Could you remind me if we changed the behaviour where a PostgreSQL source reported being `stalled` if there were no more entries in the WAL? Need to think about how to phrase this.

Reply:
Answering my own question: no ([#16786](https://github.com/MaterializeInc/materialize/issues/16786)).